packages/playground/src/weblets/tf_nextcloud.vue

```diff
@@ -0,0 +1,252 @@
+<template>
+  <weblet-layout
+    ref="layout"
+    @mount="layoutMount"
+    :cpu="solution?.cpu"
+    :memory="solution?.memory"
+    :disk="disks.reduce((total, disk) => total + disk.size, rootFilesystemSize)"
+    :ipv4="ipv4"
+    :certified="certified"
+    :dedicated="dedicated"
+    title-image="images/icons/nextcloud.png"
+  >
+    <template #title>Deploy Nextcloud All-in-One Instance </template>
+
+    <d-tabs :tabs="[{ title: 'Config', value: 'config' }]" ref="tabs">
+      <template #config>
+        <input-validator
+          :value="name"
+          :rules="[
+            validators.required('Name is required.'),
+            validators.isLowercase('Name should consist of lowercase letters only.'),
+            validators.isAlphanumeric('Name should consist of alphabets & numbers only.'),
+            name => validators.isAlpha('Name must start with alphabet char.')(name[0]),
+            validators.minLength('Name must be at least 2 characters.', 2),
+            validators.maxLength('Name cannot exceed 15 characters.', 15),
+          ]"
+          #="{ props }"
+        >
+          <input-tooltip tooltip="Instance name.">
+            <v-text-field label="Name" v-model="name" v-bind="props" />
+          </input-tooltip>
+        </input-validator>
+
+        <SelectVmImage :images="images" v-model="flist" />
+        <SelectSolutionFlavor
+          :minimum="{ cpu: 2, memory: 1024 * 4, disk: 50 }"
+          :standard="{ cpu: 4, memory: 1024 * 8, disk: 500 }"
+          :recommended="{ cpu: 4, memory: 1024 * 16, disk: 1000 }"
+          v-model="solution"
+        />
+
+        <Network
+          required
+          v-model:ipv4="ipv4"
+          v-model:planetary="planetary"
+          v-model:wireguard="wireguard"
+          ref="network"
+        />
+        <input-tooltip
+          inline
+          tooltip="Click to know more about dedicated nodes."
+          href="https://manual.grid.tf/dashboard/portal/dashboard_portal_dedicated_nodes.html"
+        >
+          <v-switch color="primary" inset label="Dedicated" v-model="dedicated" hide-details />
+        </input-tooltip>
+
+        <input-tooltip inline tooltip="Renting capacity on certified nodes is charged 25% extra.">
+          <v-switch color="primary" inset label="Certified" v-model="certified" hide-details />
+        </input-tooltip>
+
+        <SelectFarmManager>
+          <SelectFarm
+            :filters="{
+              cpu: solution?.cpu,
+              memory: solution?.memory,
+              publicIp: ipv4,
+              ssd: disks.reduce((total, disk) => total + disk.size, rootFilesystemSize),
+              rentedBy: dedicated ? profileManager.profile?.twinId : undefined,
+              certified: certified,
+            }"
+            v-model="farm"
+          />
```

Comment:
```suggestion
        <SelectFarmManager>
          <SelectFarm
            :filters="{
              cpu: solution?.cpu,
              memory: solution?.memory,
              publicIp: ipv4,
              ssd: rootFilesystemSize + (solution?.disk ?? 0),
              rentedBy: dedicated ? profileManager.profile?.twinId : undefined,
              certified: certified,
            }"
            v-model="farm"
          />
```

There is no disks to manage in this weblet

Reply:
Just to be sure I understand correctly: when I deploy [the Nextcloud FList](https://github.com/threefoldtech/tf-images/tree/development_nextcloudaio/tfgrid3/nextcloud) on the playground, I set 8gb of root file system for the nextcloud's packages, and I add a disk for the remaining space for Nextcloud storage.

In this case, aren't there the storage filesystem + the storage disk implied?